.pfnci/linux/tests/cuda-rapids.Dockerfile

```diff
@@ -1,5 +1,4 @@
-# TODO(kmaehashi): Fix stable image after release (#6805)
-ARG BASE_IMAGE="rapidsai/rapidsai-core-dev-nightly:22.08-cuda11.5-devel-ubuntu20.04-py3.8"
+ARG BASE_IMAGE="rapidsai/rapidsai-core-dev:23.06-cuda11.8-devel-ubuntu22.04-py3.10"
```

Comment:
I think that this image is not working well :(

Reply:
https://github.com/rapidsai/cugraph/pull/2866

This PR sseems to have changed the API of `weakly_connected_components`